generator.rb

```diff
@@ -0,0 +1,73 @@
+def is_vowel?(letter)
+  vowels = ["a", "e", "i", "o", "u"]
+  if vowels.include?(letter.downcase)
+    return true
+  end
+  return false
+end
+
+
+def get_word_from_user(word)
+  puts "What is the #{word} to combine?"
+  word = gets.chomp.to_s
```

Comment:
I would also advise against reusing this variable again for a different purpose! Part of your job is to convey the semantic meaning of your code with variable names, so using `word` to mean both the cardinality of the word (first second etc.) _and_ the word that we fetch from the user is confusing!